spec.html

```diff
@@ -452,37 +453,35 @@ <h1>Evaluate ( ) Concrete Method</h1>
       1. Let _result_ be InnerModuleEvaluation(_module_, _stack_, 0).
       1. If _result_ is an abrupt completion, then
         1. For each Cyclic Module Record _m_ in _stack_, do
-          1. Assert: _m_.[[Status]] is `"evaluating"`.
-          1. Set _m_.[[Status]] to `"evaluated"`.
+          1. Assert: _m_.[[Status]] is ~evaluating~.
+          1. Set _m_.[[Status]] to ~evaluated~.
           1. Set _m_.[[EvaluationError]] to _result_.
-        1. Assert: _module_.[[Status]] is `"evaluated"` and _module_.[[EvaluationError]] is _result_.
+        1. Assert: _module_.[[Status]] is ~evaluated~ and _module_.[[EvaluationError]] is _result_.
         1. <del>Return _result_.</del>
         1. <ins>Perform ! Call(_capability_.[[Reject]], *undefined*, &laquo;_result_.[[Value]]&raquo;).</ins>
       1. <ins>Otherwise,</ins>
-        1. Assert: _module_.[[Status]] is `"evaluated"` and _module_.[[EvaluationError]] is *undefined*.
+      <ins>  </ins>1. Assert: _module_.[[Status]] is ~evaluated~ and _module_.[[EvaluationError]] is *undefined*.
         1. <ins>If _module_.[[AsyncEvaluating]] is *false*, then</ins>
-          1. <ins>Perform ! Call(_capability_.[[Resolve]], *undefined*, &laquo;*undefined*&raquo;).</ins>
-        1. Assert: _stack_ is empty.
+        1. <ins>Perform ! Call(_capability_.[[Resolve]], *undefined*, &laquo;*undefined*&raquo;).</ins>
+      <ins>  </ins>1. Assert: _stack_ is empty.
       1. Return <del>*undefined*</del><ins>_capability_.[[Promise]]</ins>.
     </emu-alg>
 
     <emu-clause id="sec-innermoduleevaluation" aoid="InnerModuleEvaluation">
-      <h1>InnerModuleEvaluation( _module_, _stack_, _index_ )</h1>
-
-      <p>The InnerModuleEvaluation abstract operation is used by Evaluate to perform the actual evaluation process for the Cyclic Module Record _module_, as well as recursively on all other modules in the dependency graph. The _stack_ and _index_ parameters, as well as _module_'s [[DFSIndex]] and [[DFSAncestoreIndex]] fields, are used the same way as in InnerModuleLinking.</p>
+      <h1>InnerModuleEvaluation ( _module_, _stack_, _index_ )</h1>
+      <p>The abstract operation InnerModuleEvaluation takes arguments _module_ (a Source Text Module Record), _stack_, and _index_. It is used by Evaluate to perform the actual evaluation process for _module_, as well as recursively on all other modules in the dependency graph. The _stack_ and _index_ parameters, as well as _module_'s [[DFSIndex]] and [[DFSAncestorIndex]] fields, are used the same way as in InnerModuleLinking. It performs the following steps when called:</p>
```

Comment:
Upstream bug, it seems: _module_ need not be a **Source Text** Module Record.

Reply:
Should we do the change there first?

Reply:
https://github.com/tc39/ecma262/pull/2092